docs/modules/deployment/pages/core/system-requirements.adoc

```diff
@@ -2,51 +2,89 @@
 [[system-requirements-core]]
 = Minimum System Requirements
 
-[options="header, autowidth"]
+The following hardware specs are the minimum required to run {page-component-title}:
+
+[options="autowidth"]
 |===
-|                        | Just testing*           | Minimum server specification**
-| *CPU*                  | 2 GHz dual core x86_64  | 3 GHz quad core x86_64 and above
-| *RAM*                  | 4 GB (physical)         | 16 GB (physical) and above
-| *Storage (disk space)* | 50 GB HDD, SSD          | 1 TB with SSD and above
+|   | Just Testing^1^   | Minimum Server Specification^2^
+
+| *CPU*
+| 2GHz dual core x86_64
+| 3GHz quad core x86_64 and above
+
+| *RAM*
+| 4GB (physical)
+| 16GB (physical) and above
+
+| *Storage (disk space)*
+| 50-GB HDD, SSD
+| 1TB with SSD and above
 |===
 
-*You can install the packages and the services start. +
-**Your intended workload (network size, number of monitored metrics, flows, events, and data retention requirements) affects these requirements.
+^1^ You can install the packages, and the {page-component-title} services start. +
+^2^ Your intended workload (network size, number of monitored metrics, flows, events, and data retention requirements) affects these requirements.
```

Comment:
Table clean up (the `s|` means make this cell bold) and making the number in the table notes an ordered list as it looked odd using the superscript again.  I tested using footnote notation, but those are placed at the very bottom of the page, which doesn't look as good as directly under the table.
```suggestion
[options="autowidth"]
[cols="1,2,2"]
|===
|
s| Just Testing^1^
s| Minimum Production Specification^2^

s| CPU
| 2GHz dual core x86_64
| 3GHz quad core x86_64 or above

s| RAM
| 4GB (physical)
| 16GB (physical) or more

s| Storage
| 50GB HDD/SSD
| 1TB with SSD or more
|===

. You can install the packages, and the {page-component-title} services start.
. Your intended workload (network size, number of monitored metrics, flows, events, and data retention requirements) affects these requirements.
```

Reply:
Above renders like this on my system:
![image](https://user-images.githubusercontent.com/26419758/210676244-2de1966f-212b-4952-95e0-2d605d276296.png)

Reply:
Oh cool, I didn't know about the s| notation! I'll keep that in mind for future formatting and update these tables, but keep the header formatting as it was.